codes/javascript/chapter_computational_complexity/leetcode_two_sum.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * File: leetcode_two_sum.js
+ * Created Time: 2022-12-15
+ * Author: gyt95 (gytkwan@gmail.com)
+ */
+
+/**
+ * @param {number[]} nums
```

Comment:
We don't need this function comment, please remove it

Reply:
ok